pysrc/bytewax/connectors/bigquery/__init__.py

```diff
@@ -0,0 +1 @@
+from .bigquery_output import BigqueryOutputConfig  # noqa: F401
```

Comment:
Related: What is the thinking behind re-importing this symbol here? Now it's under both `bytewax.connectors.bigquery.BigqueryOutputConfig` and `bytewax.connectors.bigquery.bigquery_output.BigqueryOutputConfig`. It will also be duplicated in the API docs.

I think it's less confusing to have only a single canonical location. And I'd vote for `bytewax.connectors.bigquery.output.BigqueryOutputConfig`.

Not sure if this is clarifying, but: the reason we use this re-import pattern elsewhere is that Maturin doesn't let us build PyO3-based subpackages, so we have to shuffle around all the Rust-defined Python classes to their final destinations. I don't think we need that here.